recipes/picrust/meta.yaml

```diff
@@ -1,36 +1,36 @@
 {% set name = "picrust" %}
-{% set version = "1.1.3" %}
+{% set version = "1.1.4" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/picrust/picrust/archive/v{{ version }}.tar.gz
-  sha256: 7538c8544899b8855deb73a2d7a4ccac4808ff294e161530a8c8762d472d8906
+  sha256: ebfca914002ceb6d35376370e075379427d24e0c7711d46597afb52d4738c880
 
 build:
-  number: 2
-  skip: True # [ not py27 ]
+  number: 0
+  noarch: generic
 
 requirements:
   host:
-    - python
-    - setuptools
-    - biom-format >=2.1.4,<2.2.0
-    - numpy >=1.5.1
-    - cogent >=1.5.3
-    - h5py >=1.7.0
-    - future >=0.16
+    - python 2.7
+    - setuptools 41.0.1
+    - biom-format 2.1.7
+    - numpy 1.16.4
+    - cogent 1.5.3
+    - h5py 2.7.1
+    - future 0.17.1
```

Comment:
Please don't pin to exact versions either here or in the `run:` section unless absolutely needed.

Reply:
Thanks for the review @dpryan79 . Do you have any advice for how to determine the range of dependency versions that can be specified?

Reply:
For numpy don't specify anything (or specify only a minimal version), since we build everything in bioconda for a specific numpy version range. For the others, I guess the upstream package uses overly precise version pinning already, so just set whatever use things like `h5py >=2.7.1`.

Reply:
Ok thanks, good to know. The dependency version ranges have been changed (mainly back to what they were in the last recipe version).